fullcalendar-columns.js

```diff
@@ -204,7 +204,7 @@
 		},
 		_triggerExternalDrop: function(event, dropLocation, el, ev, ui) {
 			// Trigger 'drop' regardless of whether element represents an event
-			this.trigger('drop', el[0], dropLocation.start, ev, ui);
+			this.trigger('drop', el[0], dropLocation.start, ev, ui, dropLocation.column);
```

Comment:
I considered that at first, but I thought it would be less problematic to add on to the callback arguments than change the existing ones as shown in the [fullcalendar docs](https://fullcalendar.io/docs/dropping/drop/).